widgets/shortvideo/_inline-tile.scss

```diff
@@ -0,0 +1,48 @@
+@use "sass:map";
+@use "@styles/partials/media-queries";
+@use "dimens";
+
+.shortvideo-inline {
+  .ugc-tile {
+    position: relative;
+    width: 100%;
+    display: flex;
+
+    .tile {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 100%;
+
+      .tile-image-wrapper {
+        display: flex;
+        width: 100%;
+        height: var(--tile-size) !important;
+
+        img {
+          aspect-ratio: auto;
+          width: 100%;
+          object-fit: cover;
+          overflow: hidden;
+          border-radius: dimens.$inline-tile-border-radius;
+          position: relative;
+          opacity: 1;
+        }
+      }
+    }
+  }
+
+  .icon-play {
```

Comment:
Shall we use the icon from the global `icon.scss`?

Reply:
This icon is not needed in shortvideo, I will remove it